docs/reference/concepts/entity-templates.md

```diff
@@ -1,67 +1,62 @@
+  * [MonoBehaviour and ECS workflows]({{urlRoot}}/reference/workflows/which-workflow)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+  * [Standard schema library components](https://docs.improbable.io/reference/latest/shared/glossary#standard-schema-library-components)
+")%>
 
-For information on how to create SpatialOS entities once you have set up entity templates, see the documentation on [creating entities (MonoBehaviour workflow)]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/create-delete-spatialos-entities) and [ECS World commands (ECS workflow)]({{urlRoot}}/reference/workflows/ecs/interaction/commands/world-commands).
+An [`EntityTemplate`]({{urlRoot}}/api/core/entity-template) specifies what [components]({{urlRoot}}/reference/glossary#spatialos-component) a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) will have, the initial values of these components and which [layers](https://docs.improbable.io/reference/latest/shared/glossary#layers) get [write access]({{urlRoot}}/reference//glossary#authority) to each component.
 
-## How to create a SpatialOS entity template
+## How to define an entity template
 
-You have to create an `EntityTemplate` to specify which components a [SpatialOS entity]({{urlRoot}}/reference/glossary#spatialos-entity) has and the initial values of those [components]({{urlRoot}}/reference/glossary#spatialos-component). You have to also specify which type of workers can have write access (also known as ”[authority]({{urlRoot}}/reference/glossary#authority)”) on a per-component basis.
+To define an entity template, you must first construct an `EntityTemplate`.
 
-You use the `EntityTemplate` class.
+```csharp
+var entityTemplate = new EntityTemplate();
+```
 
-There are examples of how to use this class below.
+An `EntityTemplate` can be mutated and used multiple times.
 
+### Add components to entity template
 
-### Create component snapshots
-For each [schema component]({{urlRoot}}/reference/glossary#schema) you define, the `code generator` creates a struct which inherits from `ISpatialComponentSnapshot`, the generated [snapshot struct]({{urlRoot}}/reference/concepts/code-generation#snapshot). For example, for the following schema:
+All SpatialOS entities require the `Position` and `EntityAcl` components.
 
-```
-component Health {
-  id = 1;
-  int32 current_health = 1;
-}
-```
+The `Position` component must be added to the entity template manually. It is used by SpatialOS for load-balancing purposes and some types of queries.
```

Comment:
do we want to be more specific about what kind of queries?

Reply:
Ah I forgot to link this to a couple of other docs pages

I think we could give an example e.g. relative constraints